pkg/app/web/src/components/generated-api-key-dialog.tsx

```diff
@@ -12,38 +12,39 @@ import {
 import CopyIcon from "@material-ui/icons/FileCopyOutlined";
 import copy from "copy-to-clipboard";
 import { addToast } from "../modules/toasts";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
+import { AppState } from "../modules";
+import { clearGeneratedKey } from "../modules/api-keys";
 
 const useStyles = makeStyles(() => ({
   key: {
     wordBreak: "break-all",
   },
 }));
 
-interface Props {
-  open: boolean;
-  generatedKey: string | null;
-  onClose: () => void;
-}
-
 const DIALOG_TITLE = "Generated API Key";
 const VALUE_CAPTION = "API Key";
+const TOAST_MESSAGE = "API Key copied to clipboard";
```

Comment:
should move to `pkg/app/web/src/constants/toast-text.ts` 👀

Reply:
Yeah, exactly! Thanks!